modules/src/lib.rs

```diff
@@ -15,9 +15,11 @@
 //!
 //! - ICS 02: Client
 //! - ICS 03: Connection
+//! - ICS 04: Channel
 //! - ICS 07: Tendermint Client
 //! - ICS 23: Vector Commitment Scheme
```

Comment:
add ICS 20?

Reply:
I hesitated as it was in a very minimal state. But good point, it's better to document it here regardless.